### DISCUSSION TITLE
Font setting and row height?

### DISCUSSION THREAD

**Discussion by rtoi** [lvjr/tabularray#244] [Q&A]
I have three identical tables except for font size settings.
The first table is added to the environment where the required font size is set. 
For the second table the font size is set by  `font` key of `rows` option.  
Third table is for reference without any font size settings.

```latex
\documentclass[a5paper]{article}
\usepackage{tabularray}

\begin{document}
  Table 1\hfil Table 2\hfil Table 3
  
  {
    \fontsize{7.5}{8}\selectfont
    \begin{tblr}{colspec={ll},hlines}
      Aaaa & Bbbb\\
      Dddd & Eeee\\
      Gggg & Hiii
    \end{tblr}
  }\hfil
  \begin{tblr}{colspec={ll},hlines,
      rows={font=\fontsize{7.5}{8}}}
    Aaaa & Bbbb\\
    Dddd & Eeee\\
    Gggg & Hiii
  \end{tblr}\hfil
  \begin{tblr}{colspec={ll},hlines}
    Aaaa & Bbbb\\
    Dddd & Eeee\\
    Gggg & Hiii
  \end{tblr}
  
\end{document}  
```

I expected tables 1 and 2 look the same with same font size and row height. However only table 1 shows the expected row height. Apparently, the row height of table 2 is calculated based on the default font, as in table 3. 

![tblr-font-size-row-height](https://user-images.githubusercontent.com/3971976/167387944-b01cf051-493e-472b-85ce-4a8e47af9e93.png)

Is this the expected behaviour?

**Comment by lvjr** [comment]
Yes, it is  an expected behavior. I am planning to add `font` option for tables in the future, then you will be able to write
```latex
\begin{tblr}{font=\fontsize{7.5}{8}}
  Aaaa & Bbbb \\
\end{tblr}
```
to get the result of Table 1.

At this time, you can also use `\SetTblrInner{stretch=0}` to make Table 1 and 2 produce the same result.

**Comment by edilibre** [comment]
Hi,
is there a chance that the font feature for setting the fontsize for a whole table is available nowadays?
Thanks!
